src/rgw/driver/rados/cls_fifo_legacy.cc

```diff
@@ -1292,6 +1313,19 @@ void FIFO::read_meta(const DoutPrefixProvider *dpp, std::uint64_t tid, lr::AioCo
   assert(r >= 0);
 }
 
+void FIFO::maybe_backoff(const DoutPrefixProvider* dpp, std::uint64_t tid,
```

Comment:
this function can be stanalone, doesn't use any variables from Fifo class !

Reply:
It uses `get_backoff()`.